powerline/segments/common/bat.py

```diff
@@ -189,6 +174,8 @@ def battery(pl, format='{capacity:3.0%}', steps=5, gamify=False, full_heart='O',
 		another gradient level and highlighting group, so it is OK for it to be 
 		the same as full_heart as long as necessary highlighting groups are 
 		defined.
+	:param str charging:
+		Indication of "AC charging"
```

Comment:
This is missing a dot. Also it should be more explicit about what kind of indication it is: “If computer is connected to a power supply this symbol is prepended to the segment.”.